### DISCUSSION TITLE
The future of css-modules in Next.js

### DISCUSSION THREAD

**Discussion by uny** [vercel/next.js#17999] [Help]
Seeing the comment [1], I wonder if I should use css-modules from now for Next.js.
On the other hand, I also saw that css-modules is recommended over css-in-js [2].

Won't css-modules be deprecated for Next.js and Can I use it without worries about the future?

[1] https://github.com/webpack-contrib/css-loader/issues/1050#issuecomment-592541379
[2] https://github.com/vercel/next.js/issues/15542

**Comment by matamatanot** [comment]
I am also curious about this topic. 

My team has basically adopted the methods which official Next.js recommend. However, I am a little skeptical about the future of CSS Modules.
